commit a7395cadf27a69889b3c5acb6d39491a2e76a1f6 (refs/changes/19/2519/1)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-09-08T13:00:36+00:00 (2 years, 1 month ago)
    
    OS-6320 systemd aborts itself on suse image [suppress lint warning]

diff --git a/usr/src/uts/common/brand/lx/io/lx_netlink.c b/usr/src/uts/common/brand/lx/io/lx_netlink.c
index e1f17cef59..2c8a33bfed 100644
--- a/usr/src/uts/common/brand/lx/io/lx_netlink.c
+++ b/usr/src/uts/common/brand/lx/io/lx_netlink.c
@@ -513,6 +513,7 @@ lx_netlink_setsockopt(sock_lower_handle_t handle, int level,
 	}
 }
 
+/*ARGSUSED*/
 static int
 lx_netlink_getsockopt(sock_lower_handle_t handle, int level,
     int option_name, void *optval, socklen_t *optlen, cred_t *cr)
